irohad/ametsuchi/impl/rocksdb_common.hpp

```diff
@@ -931,14 +981,14 @@ namespace iroha::ametsuchi {
       if (status.assumeValue().ok()) {
         auto const &[major, minor, patch] =
             staticSplitId<3ull>(common.valueBuffer(), "#");
-        IrohadVersion version;
+        IrohadVersion version{0ul, 0ul, 0ul};
```

Comment:
Please use some huge value or even better negative. It will show that value from database was not read.

Currently we will see 0 which is real usable value even if database provides wrong value.

Reply:
It doesn't matter. This initialization just to close warnings. There is no way to leave such a values in this object.

Reply:
std::from_chars could fail and you do not check the result.

Reply:
It seems to be redundant ```std::from_chars``` error check here. I don't care about it. We have the version directly used in our code. It doesn't matter what happens(no version, incorrect version, trash) while reading. All of these results to incorrect version error generation. And it doesn't matter that 2nd character is not equal of expected. The error and the following workflow will be the same. It also doesn't matter for the end-user, because we don't need him to use database manually.